config/services.yaml

```diff
@@ -41,10 +41,9 @@ services:
         tags:
             - { name: twig.extension }
 
-    App\Command\FetchConferencesCommand:
+    App\Fetcher\ConferencesHarvester:
         arguments:
-            - !tagged app.fetcher
-            - '@doctrine'
+            $fetchers: !tagged app.fetcher
```

Comment:
`!$tagged` is deprecated IIRC, you must use `tagged_iterator` or `tagged_locator` (here you want the iterator)